niri-ipc/src/lib.rs

```diff
@@ -214,6 +214,9 @@ pub enum Action {
         /// Id of the window to focus.
         #[cfg_attr(feature = "clap", arg(long))]
         id: u64,
+        /// Dont move the cursor when focussing the window.
+        #[cfg_attr(feature = "clap", arg(long = "no_mouse_warp", action = clap::ArgAction::SetTrue))]
```

Comment:
Why spell out the name in `long = "no_mouse_warp"`? Also, is `SetTrue` needed even for bools?